controls/C-0239-avoiduseofsystemmastersgroup.json

```diff
@@ -0,0 +1,20 @@
+{
+    "controlID": "C-0239",
```

Comment:
This controlID has been used in cis-aks, need to change to unique controlID.
We need to think of a better way to handle this, it'll be difficult to keep track..

Reply:
going to use `C-0246`.